stdlib/2and3/turtle.pyi

```diff
@@ -0,0 +1,478 @@
+from typing import Tuple, overload, Optional, Union, Dict, Any, Sequence, TypeVar, List, Callable
+import sys
+if sys.version_info >= (3,):
+    from tkinter import Canvas
+else:
+    # TODO: Replace this with 'from Tkinter import Canvas' once we add stubs for Tkinter
+    Canvas = Any
+
+Color = Union[str, Tuple[float, float, float]]
+Speed = Union[str, int]
+
+# TODO: Type this more accurately
+# Vec2D is actually a custom subclass of 'tuple'.
+Vec2D = Tuple[float, float]
+
+class TurtleScreenBase:
+    cv: Canvas = ...
+    canvwidth: int = ...
+    canvheight: int = ...
+    xscale: float = ...
+    def __init__(self, cv: Canvas) -> None: ...
+    if sys.version_info >= (3,):
+        def mainloop(self) -> None: ...
+        def textinput(self, title: str, prompt: str) -> Optional[str]: ...
+        def numinput(self, title: str, prompt: str, default: Optional[float] = ..., minval: Optional[float] = ..., maxval: Optional[float] = ...) -> Optional[float]: ...
+
+class Terminator(Exception): ...
+class TurtleGraphicsError(Exception): ...
+
+class Shape:
+    def __init__(self, type_: str, data: Optional[Sequence[Tuple[float, float]]] = ...) -> None: ...
+    def addcomponent(self, poly: Sequence[Tuple[float, float]], fill: Color, outline: Optional[Color] = ...) -> None: ...
+
+class TurtleScreen(TurtleScreenBase):
+    def __init__(self, cv: Canvas, mode: str = ..., colormode: float = ..., deplay: int = ...) -> None: ...
+    def clear(self) -> None: ...
+    @overload
+    def mode(self) -> str: ...
+    @overload
+    def mode(self, mode: str) -> None: ...
+    def setworldcoordinates(self, llx: float, lly: float, urx: float, ury: float) -> None: ...
+    def register_shape(self, name: str, shape: Union[Sequence[Tuple[float, float]], Shape, None] = ...) -> None: ...
+    @overload
+    def colormode(self) -> float: ...
+    @overload
+    def colormode(self, cmode: float) -> None: ...
+    def reset(self) -> None: ...
+    def turtles(self) -> List[Turtle]: ...
+    @overload
+    def bgcolor(self) -> str: ...
+    @overload
+    def bgcolor(self, color: Color) -> None: ...
+    @overload
+    def bgcolor(self, r: float, g: float, b: float) -> None: ...
```

Comment:
Should be a triple of floats.

Reply:
That should be handled by the second overload -- `_Color` is defined to accept a tuple of three floats. (I also double checked, and it looks like all of the color functions support the three float argument form.)

Reply:
True, I misread the code.